runtime/src/bank.rs

```diff
@@ -3810,13 +3818,13 @@ impl Bank {
     ///     - partition_index: reward partition index
     /// Store the rewards to AccountsDB, update reward history record and total capital.
     fn credit_epoch_rewards_in_partition(&mut self, partition_index: u64) {
-        if let EpochRewardStatus::Active(StartSlotBlockHeightAndRewards {
-            parent_start_slot: _parent_start_slot,
+        if let EpochRewardStatus::Active(StartBlockHeightAndRewards {
             parent_start_block_height: _parent_start_block_height,
             calculated_epoch_stake_rewards: ref stake_rewards,
         }) = self.epoch_reward_status
         {
             let mut metrics = RewardsStoreMetrics {
+                // jwash: should we include cap in the metrics?
```

Comment:
@HaoranYi Ha. I don't remember exactly what I was asking. I guess I'm asking if we SHOULD be putting capitalization in the metrics at all. We could info log it, but in the metrics? I guess that is ok.
I can just remove my comment.